src/lib.rs

```diff
@@ -168,6 +201,7 @@ impl Default for Conf<'_> {
             p2p: P2P::No,
             network: "regtest",
             tmpdir: None,
+            datadir: None,
```

Comment:
There is a doctest for `Conf` struct where default values are documented, could you add a line `conf.datadir = None;`

Reply:
Done..